x/claims/types/claims_record.go

```diff
@@ -7,6 +7,9 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+// IBCTriggerAmt is the amount required to trigger a merge/migration of claims records
+var IBCTriggerAmt = sdk.NewInt(3940000000)
```

Comment:
Why this number?
This should be less than 6 digits

Reply:
It's an arbitrary number. The amount should not be transferred anyways. Happy to change it though